Lib/multiprocessing/connection.py

```diff
@@ -73,6 +73,11 @@ def arbitrary_address(family):
     if family == 'AF_INET':
         return ('localhost', 0)
     elif family == 'AF_UNIX':
+        # Prefer abstract sockets if possible to avoid problems with the address
+        # size.  When coding portable applications, some implementations have
```

Comment:
"address size": are you talking about the absolute path of the UNIX socket?

Reply:
Yes